drush-services.yml

```diff
@@ -0,0 +1,42 @@
+#
+# Drush services
```

Comment:
We do not support any user editing of this file, but we could (and probably eventually will) add a feature to have Drush load additional service files from other locations.